ios/FluentUI/Bottom Sheet/BottomSheetController.swift

```diff
@@ -316,6 +318,30 @@ public class BottomSheetController: UIViewController, Shadowable, TokenizedContr
         }
     }
 
+    /// Changes the `isHidden` state with a completion handler.
+    /// - Parameters:
+    ///   - isHidden: The new value.
+    ///   - unhiddenTargetState: The state the sheet should expand to when not hidden.
+    ///   - animated: Indicates if the change should be animated. The default value is `true`.
+    ///   - completion: Closure to be called when the state change completes.
+    @objc public func setHidden(_ isHidden: Bool, unhiddenTargetState: BottomSheetExpansionState = .collapsed, animated: Bool = true, completion: ((_ isFinished: Bool) -> Void)? = nil) {
+        let targetState: BottomSheetExpansionState
+        if unhiddenTargetState != .collapsed && unhiddenTargetState != .expanded {
+            targetState = isHidden ? .hidden : .collapsed
```

Comment:
To simplify, I'm tempted to just use `unhiddenTargetState` if isHidden is false. At least that way we do what the caller asked, even if that doesn't make sense. What do you think?

Reply:
I am not sure I follow exactly what you are proposing? When the caller calls `setHidden(true)` they would always go to the hidden state regardless of the value of `unhiddenTargetState`.

Reply:
You have special logic here which handles unhiddenTargetState set to .hidden. I'm proposing we don't have a special case for that.

Reply:
Oh the logic behind the protection is actually to prevent users from using `.transitioning`. Why anyone would want to do that is beyond me, but I figure we should handle it.